packages/eslint-plugin/ROADMAP.md

```diff
@@ -59,6 +59,7 @@
 | [`no-empty`]                         | 🌟  | [`no-empty`][no-empty]                                                |
 | [`no-eval`]                          | 🌟  | [`no-eval`][no-eval]                                                  |
 | [`no-floating-promises`]             | 🛑  | N/A ([relevant plugin][plugin:promise])                               |
+| [`no-for-in-array`]                  | ✅  | [`typescript/no-for-in-array`]                                        |
```

Comment:
```suggestion
| [`no-for-in-array`]                  | ✅  | [`@typescript-eslint/no-for-in-array`]                                        |
```

This should also be added to the references section at the bottom of the document so the link works.

Reply:
Added.